src/common/Screen.js

```diff
@@ -46,19 +47,21 @@ class Screen extends PureComponent<Props> {
   };
 
   static defaultProps = {
-    scrollView: true,
     centerContent: false,
+    keyboardShouldPersistTaps: 'never',
+    scrollView: true,
```

Comment:
Huh, looks like this `scrollView` prop isn't actually used. Am I missing how it is?

If it really isn't, it'd be a nice further cleanup (in a separate commit, of course, and doesn't have to be this PR) to remove it.